Cargo.toml

```diff
@@ -45,17 +45,17 @@ serde_json = "1.0"
 toml = "0.4.0"
 
 [dependencies.sass-rs]
-version = "0.2.0"
+version = "0.2"
 optional = true
 
 [dependencies.hyper]
 version = "0.10"
 default-features = false
 
 [dependencies.syntect]
-version = "1.8.0"
+version = "1.8"
 optional = true
-default-features = false
+default-features = true
```

Comment:
Is there a reason you are enabling the default features of `syntect`?

Reply:
I admit it was kind of trials and errors here ^^' Not sure this one is needed.

Reply:
Thanks